test/end-to-end/fiscalYears/fiscalYears.spec.js

```diff
@@ -75,4 +75,88 @@ describe('Fiscal Year', function () {
     components.modalAction.confirm();
     components.notification.hasSuccess();
   });
+
+  it('set the opening balance for the first fiscal year', () => {
+    helpers.navigate(path);
```

Comment:
I think it automatically navigates in a `before()` block.  I could be wrong though.